Source/RxCocoa/UIScrollView+reachedBottom.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UIScrollView+reachedBottom.swift
+//  RxSwiftExt
+//
+//  Created by Anton Nazarov on 09/05/2019.
+//  Copyright © 2019 RxSwift Community. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+public extension Reactive where Base: UIScrollView {
+    /**
+     Shows if the the bottom of the UIScrollView is reached.
+     - parameter offset: A threshhold indicating the bottom.
+     - returns: Sequence that emits when the bottom of the base UIScrollView is reached.
```

Comment:
suggestion: `Single` that emits when the bottom of the `UIScrollView` is reached.

Reply:
It's not a Single

Reply:
Damn! Totally overlooked it. I thought we where talking about Singles.